source/DataBase/DataBaseCommon.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <iostream>
+#include <vector>
+
+struct DataBaseCommon
+{
+    static const std::vector<std::string> userRecords;
+    static const std::vector<std::string> passwordRecords;
```

Comment:
Why do we need these vectors? Where are they being used?

Reply:
These vectors contain Table parameters from the databases. They are useful for using the insert method. Will update this with comments regarding changes to the database